app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,27 @@
+class UserMailer < ApplicationMailer
+  before_action :set_user
+  before_action :set_event_user
+  
+  def welcome_email(user)  
+    @url  = 'http://example.com/login'
+    mail(to: @user.email, subject: 'Welcome to My Awesome Site')
+  end
+
+  def created_user
+    mail(to: @user.email, subject: "[DIGITAL FLIGHT SIM] New User Created: #{@event_user.email}")
+  end
+
+  def update_user    
+    mail(to: @user.email, subject: "[DIGITAL FLIGHT SIM] New Aircraft Created: #{@event_user}")
+  end
+
+  private
+
+  def set_user
+    @user = params[:user]
```

Comment:
```suggestion
    @user = params[:recipient]
```

parameter for mailer should be `recipient`